kratos/utilities/volume_inside_voxel_utility.h

```diff
@@ -34,6 +34,17 @@ namespace Kratos
 ///@name Type Definitions
 ///@{
 
+/**
+ * @namespace VolumeInsideVoxedDefinition
+ * @ingroup KratosCore
+ * @brief Neighbours definition
+ * @author Ariadna Cortes
+ */
+namespace VolumeInsideVoxedDefinition
+{
+    static constexpr std::size_t Neighbours[4][2] = {{3,1},{0,2},{1,3},{2,0}};
+}
```

Comment:
I meant a `constexpr` inside the class. You don't want to make this available for the entire Kratos.

Reply:
That was my proposal to solve a compilation issue with Unity. I tried to do the definition in the CPP, but it didn't work. I will try again. Sorry for the noise.

Reply:
Okay reverted, apparently last time when I tried I didn't save or something, but now it works